vpdq/MANIFEST.in

```diff
@@ -0,0 +1,16 @@
+include pdq/cpp/common/pdqhashtypes.cpp
```

Comment:
It seems like this manifest uses the parent directory as the base directory. Is that intentional?

Can we use the current 'vpdq' directory? Relying on the current directory having a certain name reduces portability of the directory.

Reply:
Yes, it is intentional. I want to inlude files in pdq directory where it is parallel to vpdq directory. So, I have to do this in parent directory.